### PR TITLE
Fix create of preloadedState when no React Dev Tools extension is pre…

### PR DESCRIPTION
**Pull request by milenvoutchev**
…sent

### CODE REVIEW

**Comment by artemstepanenko** [comment]
Looks fine. Waiting for a verdict from @gkatsanos .

File: src/index.js [hunk 1]
```diff
@@ -9,14 +9,12 @@ import App from './App';
...
 
-const store = createStore(
-  reducer,
-  compose(
-    applyMiddleware(thunk),
-    window.__REDUX_DEVTOOLS_EXTENSION__ &&
-      window.__REDUX_DEVTOOLS_EXTENSION__(),
-  ),
-);
+const preloadedState = window.__REDUX_DEVTOOLS_EXTENSION__ &&
```
Comment (tobias-reuter-rocket) [inline src/index.js]: The correct parameter name is 'enhancer' and not 'preloadedState'. Please check the docs and correct it accordingly.

**Comment by gkatsanos** [comment]
I think that since we use `process.env.NODE_ENV === 'development'` , it's implied that a developer who will clone this needs to have the Redux Dev tools, although I agree the error could be handled more gracefully

File: src/index.js (hunk 1 above)
Comment (milenvoutchev) [inline src/index.js]: It says `preloadedState` on the version I look at. Maybe I'm missing something, but here it is:
https://github.com/reactjs/redux/blob/master/docs/api/createStore.md

Comment (milenvoutchev) [inline src/index.js]: Ah ok, got it. Second and parameters are optional. Will change it, thanks.

**Comment by milenvoutchev** [comment]
@tobias-reuter-rocket Changed the name of the param properly reflect its function.

File: src/index.js [hunk 2]
```diff
@@ -9,14 +9,12 @@ import App from './App';
...
 
-const store = createStore(
-  reducer,
-  compose(
-    applyMiddleware(thunk),
-    window.__REDUX_DEVTOOLS_EXTENSION__ &&
-      window.__REDUX_DEVTOOLS_EXTENSION__(),
-  ),
-);
+const enhancer = window.__REDUX_DEVTOOLS_EXTENSION__ &&
```
Comment (tobias-reuter-rocket) [inline src/index.js]: How about:
```
let storeEnhancer = [applyMiddleware(thunk)];

if (window.__REDUX_DEVTOOLS_EXTENSION__ &&
    process.env.NODE_ENV === 'development') {
  storeEnhancer = [...storeEnhancer, window.__REDUX_DEVTOOLS_EXTENSION__()];
}

const store = createStore(reducer, compose(...storeEnhancer));
```
?

It let us add and remove enhancer without changing a lot of lines.
I also think it is easier to read, as the first and default enhancer is in the first line and every additional enhancer gets their own lines.

**Pull request merged**